src/help/plugin.ts

```diff
@@ -149,19 +149,21 @@ function activate(app: JupyterLab, mainMenu: IMainMenu, palette: ICommandPalette
   // Handle state restoration.
   restorer.restore(tracker, {
     command,
-    args: widget => ({ isHidden: widget.isHidden, url: widget.url }),
-    name: widget => namespace
+    args: widget => ({ url: widget.url }),
```

Comment:
This needs to be: `args: widget => ({ url: widget.url, text: widget.title.label }),` to preserve the title.